molgenis-data-elasticsearch/src/main/java/org/molgenis/data/elasticsearch/client/ClientFacade.java

```diff
@@ -522,6 +522,7 @@ public Explanation explain(SearchHit searchHit, QueryBuilder query)
 		}
 
 		String indexName = searchHit.getIndex();
+		//FIXME: ClientFacade shouldn't assume that typename equals typename
```

Comment:
Did you add the FIXME's and TODO's in this file to the indexing document on Google Drive?